### ISSUE TITLE
[Contracts] create automated service to update database from changes in contracts.

### ISSUE THREAD

**Issue by pelsasser** [MARKETProtocol/market-api#29]
## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

### User Story
see https://github.com/MARKETProtocol/market-api/issues/28

### Why Is this Needed?
*Summary*: to update the dynamic fields that are housed in our database and add new contracts periodically

### Description
*Type*: Feature

### Solution
*Summary*: Implement a service that provides the needed functionality to keep our database up to date.

### Definition of Done
- listen for events `AddressAddedToWhitelist` and  `AddressRemovedFromWhitelist` and add / remove contracts
- Listen to events on contract list to update the following dynamic fields
  - Is Settled
  - Collateral Pool Balance
  - Last Traded Price
  - Last Queried Price

### Additional Information
depends on #28 

### TODO: explanation of app creation.
